setup/rt_literal_property_attrs_doc.json

```diff
@@ -37,6 +37,32 @@
         }
       ]
     },
+    {
+      "@id": "_:b3",
+      "@type": [
+        "http://sinopia.io/vocabulary/PropertyTemplate"
+      ],
+      "http://www!w3!org/2000/01/rdf-schema#label": [
+        {
+          "@value": "Validation Regular Expression"
+        }
+      ],
+      "http://sinopia!io/vocabulary/hasRemark": [
+        {
+          "@value": "Regular Expression to validate the literal."
+        }
+      ],
+      "http://sinopia!io/vocabulary/hasPropertyUri": [
+        {
+          "@id": "http://sinopia.io/vocabulary/hasValidationRegex"
```

Comment:
should this be "http://sinopia.io/vocabulary/propertyAttribute/hasValidationRegex"  (or even l"iteralAttribute")